src/slack_interaction_handler.ts

```diff
@@ -260,18 +260,12 @@ export async function handleVoterStatusUpdate({
         );
       }
 
-      // HACK: work around slack bug updating blocks by adjusting the block_id
-      payload.message.blocks[2]['block_id'] = Math.random()
-        .toString(36)
-        .replace(/[^a-z]+/g, '')
-        .substr(0, 8);
-
       // Replace the entire block so that the initial option change persists.
```

Comment:
You change `replaceSlackMessageBlocks` with `updateVoterStatusBlocks` but the comment above still says "Replace the entire block..."